lib/vtls/schannel.c

```diff
@@ -622,12 +631,7 @@ static CURLcode schannel_acquire_credential_handle(struct Curl_cfilter *cf,
         else
           pszPassword[0] = 0;
 
-        if(Curl_isVistaOrGreater)
-          cert_store = PFXImportCertStore(&datablob, pszPassword,
-                                          PKCS12_NO_PERSIST_KEY);
-        else
-          cert_store = PFXImportCertStore(&datablob, pszPassword, 0);
-
+        cert_store = PFXImportCertStore(&datablob, pszPassword, import_flags);
```

Comment:
I take issue with deleting this security feature.
Leaving secret keys on disk and later trying to 
delete them manually seem very error-prone.